packages/salesforcedx-vscode-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -281,6 +289,23 @@ export class ApexReplayDebug extends LoggingDebugSession {
             this.convertClientLineToDebugger(lineArg)
           );
         }
+        // If there is a condition and that condition is a checkpoint then
+        // sent CHECKPOINT_INFO_EVENT with the args.source, line and uri
+        if (args.breakpoints[i].condition) {
+          if (
+            args.breakpoints[i].condition!.toLowerCase().indexOf(CHECKPOINT) >=
+            0
+          ) {
+            // JRS
```

Comment:
What's this comment for?

Reply:
Gone, it was just a marker for my searching.